addons/im_livechat/demo/im_livechat_channel/im_livechat_support_bot_session_1.xml

```diff
@@ -85,5 +85,66 @@
             <field name="user_raw_script_answer_id" ref="chatbot_script_troubleshooting_answer_chrome_demo"/>
         </record>
+        <record id="support_bot_session_1_message_5" model="mail.message">
+            <field name="model">discuss.channel</field>
+            <field name="res_id" ref="support_bot_session_1_demo"/>
+            <field name="author_id" ref="support_bot_operator_partner_demo"/>
+            <field name="body">What module are you interested in?</field>
+            <field name="message_type">comment</field>
+            <field name="subtype_id" ref="mail.mt_comment"/>
+            <field name="date" eval="DateTime.today() + relativedelta(months=-1, minutes=5)"/>
+        </record>
+        <record id="support_bot_session_1_message_6" model="mail.message">
+            <field name="model">discuss.channel</field>
+            <field name="res_id" ref="support_bot_session_1_demo"/>
+            <field name="author_id"/>
+            <field name="author_guest_id" ref="support_bot_session_1_guest_demo"/>
+            <field name="body">Discuss and Livechat</field>
+            <field name="message_type">comment</field>
+            <field name="subtype_id" ref="mail.mt_comment"/>
+            <field name="date" eval="DateTime.today() + relativedelta(months=-1, minutes=6)"/>
+        </record>
+        <record id="support_bot_session_1_chatbot_message_5" model="chatbot.message">
+            <field name="mail_message_id" ref="support_bot_session_1_message_6"/>
+            <field name="discuss_channel_id" ref="support_bot_session_1_demo"/>
+            <field name="script_step_id" ref="chatbot_script_initial_step_demo"/>
+            <field name="user_script_answer_id" ref="chatbot_script_answer_livechat_demo"/>
+            <field name="user_raw_script_answer_id" ref="chatbot_script_answer_livechat_demo"/>
+        </record>
+        <record id="support_bot_session_1_message_7" model="mail.message">
+            <field name="model">discuss.channel</field>
+            <field name="res_id" ref="support_bot_session_1_demo"/>
+            <field name="message_type">comment</field>
+            <field name="author_id" ref="support_bot_operator_partner_demo"/>
+            <field name="body">Let me check if I can connect you with the right expert...</field>
+            <field name="subtype_id" ref="mail.mt_comment"/>
+            <field name="date" eval="DateTime.today() + relativedelta(months=-1, minutes=7)"/>
+        </record>
+        <record id="support_bot_session_1_member_agent_demo" model="discuss.channel.member">
+            <field name="partner_id" ref="base.partner_admin"/>
+            <field name="channel_id" ref="support_bot_session_1_demo"/>
+            <field name="livechat_member_type">agent</field>
+            <field name="agent_expertise_ids" eval="[(4, ref('im_livechat_chatbot_script_expertise_discuss_demo')),(4, ref('im_livechat_chatbot_script_expertise_livechat_demo'))]"/>
+        </record>
```

Comment:
It would be better to call `_add_members` as it is more accurate: notification posted, history created automatically.